imports/plugins/included/product-admin/client/components/TagSelector/TagSelector.js

```diff
@@ -0,0 +1,223 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import { i18next } from "/client/api";
+import CloseIcon from "mdi-material-ui/Close";
+import Select from "@reactioncommerce/catalyst/Select";
+import SplitButton from "@reactioncommerce/catalyst/SplitButton";
+import { useApolloClient } from "@apollo/react-hooks";
+import {
+  Grid,
+  Card as MuiCard,
+  CardActions,
+  CardHeader,
+  CardContent,
+  IconButton,
+  makeStyles
+} from "@material-ui/core";
+import { getTags } from "./helpers";
+import { ADD_TAGS_TO_PRODUCTS, REMOVE_TAGS_FROM_PRODUCTS } from "./mutations";
+import Notifications from "./Notifications";
+
+const ACTION_OPTIONS = [{
+  label: "Add tags to products",
+  type: "ADD"
+}, {
+  label: "Remove tags from products",
+  isDestructive: true,
+  type: "REMOVE"
+}];
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    overflow: "visible",
+    marginTop: theme.spacing(4)
+  },
+  cardContainer: {
+    alignItems: "center"
+  },
+  cardHeaderTitle: {
+    letterSpacing: "0.3px"
+  },
+  cardActions: {
+    justifyContent: "flex-end"
+  },
+  hidden: {
+    display: "none"
+  },
+  visible: {
+    display: "block"
+  },
+  helpText: {
+    marginLeft: "20px",
+    letterSpacing: "0.28px",
```

Comment:
Use literal numbers, and omit units from letter spacing and margin.

Reply:
- letterspacing should come from the theme.
- Margin should use the spacing unit if possible, and if not be a literal number.

Omit the unites of measure when possible, this allows the theme util to generate the correct one for each situation.